manim/plugins/import_plugins.py

```diff
@@ -0,0 +1,41 @@
+import pkg_resources
+from importlib import import_module
+from .. import config, logger
+
+__all__ = []
+
+# don't know what is better to do here!
+module_type = type(pkg_resources)
+function_type = type(import_module)
+
+plugins_requested: list = config["plugins"]
+if "" in plugins_requested:
+    plugins_requested.remove("")
+for plugin in pkg_resources.iter_entry_points("manim.plugins"):
+    if plugin.name in plugins_requested:
```

Comment:
Might be cleaner to instead do
```suggestion
    if plugin.name not in plugins_requested:
        continue
```
to avoid making the success condition the deeper condition, which generally helps readability.

Or perhaps you could loop over a `filter(...)` but I think that would make the code harder to read.

Reply:
Ah, that requires everything to get indented differently.

Reply:
Yeah sorry, I didn't want my suggestion to be too big for such a simple thing

Reply:
done... https://github.com/ManimCommunity/manim/pull/967/commits/e5e4463d9b43266d704b8a045bd0553198362b8d